dist/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width , initial-scale = 1.0">
+    <script defer src="main.js"></script></head>
+    <body>
+        <h1 class="header">Leaderboard</h1>
+        <div class = "container">
+        <section class="scores">
+        
+         
+              <p class="score-title"><span class="scored">Recent Scored</span><button type="button">Refresh</button></p>
+                <ul id = "list-of-score">
+                    <!-- <li>Munana: <span>20</span></li>
+                    <li>Munana: <span>20</span></li> -->
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1:

Reply:
hello @NahnahAJ, Thank you for reviewing my PR I have made the suggested changes